uwrt_mars_rover_pid_api/include/uwrt_mars_rover_pid_api.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include "uwrt_can.h"
+#include "fw_config.h"
+#include "ros/ros.h"
+
+#define PID_API_TESTING
+
+class pidApi {
+    private:
+    ros::ServiceServer _ss;
+    ros::NodeHandle _nh;
+    ros::Rate _loop_rate;
+    uwrt_utils::UWRTCANWrapper CANMsg;
+    void pidApi::updatePIDGain(uwrt_mars_rover_msgs::set_state::Request &req, uwrt_mars_rover_msgs::set_state::Response &res);
+    std::string stringifyParam(unit8_t param);
```

Comment:
maybe these should be part of the config. the getters and setters.

Reply:
should the defns be in the .h or should i make a .cpp

Reply:
idk how best to structure this. u can keep it in the header for now but keep it simple and short.